util/log/log.go

```diff
@@ -27,6 +27,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const timeFormat = "1/2 15:04:05.000" // m/d h:m:s.nanos
```

Comment:
I should have mentioned that currently it's more important to have readable log output, and there have been complaints from the team about eye cancer. We'll have decent timestamps eventually, but possibly we'll also just have logs for humans and machines separately.